crypto/src/main/java/org/web3j/crypto/transaction/type/Transaction4844.java

```diff
@@ -139,7 +139,7 @@ public List<RlpType> asRlpValues(Sign.SignatureData signatureData) {
         resultTx.add(RlpString.create(data));
 
         // access list
-        resultTx.add(new RlpList());
+        result.add(new RlpList(rlpAccessListRlp()));
```

Comment:
Are you sure you want to change it from `resultTx` to `result`?  I think it should stay in `resultTx`: https://eips.ethereum.org/EIPS/eip-4844